### DISCUSSION TITLE
Fire burning before it should

### DISCUSSION THREAD

**Discussion by dmmcmahon-lanl** [firemodels/fds#13752] [Q&A]
I am trying to set up a fire that burns for a short period at a constant HRR. For some reason FDS seems to start it burning immediately when the simulation begins at t=0. I set it up as a ramp maybe there is something wrong with my setup? I have tried starting the fire at different times including at 300 s into the simulation and it still appears to start at t=0. 

&VENT XB= 1.06,3.96,1.2,4.1,0.0, 0.0, SURF_ID='Pool Fire', XYZ= 2.51, 2.65, 0.0 /
&SURF ID='Pool Fire', RAMP_Q='OilFire', HRRPUA=1794.0, COLOR='RED' /
&RAMP ID='OilFire',        T=      0,        F=    0.00 /
RAMP ID='OilFire',          T=      25,        F=    0.00 /
RAMP ID='OilFire',          T=      30,        F=    1.00 /
RAMP ID='OilFire',          T=      50,        F=    1.00 /
RAMP ID='OilFire',          T=      55,        F=    1.00 /
RAMP ID='OilFire',          T=      60,        F=    0.00 /

**Comment by drjfloyd** [comment]
You don't have a valid RAMP. & is required for the input to be recognized. I am surprised you didn't get an error message that your RAMP only has one line. Are you using an old version?

**Comment by dmmcmahon-lanl** [comment]
Thanks for catching that, I had been staring at it so much I wasn't seeing it. I am using version 6.8.0

**Reply by drjfloyd** [reply]
With 6.9.1 it would have given the error:

ERROR(392): RAMP OilFire has only one point. (CHID: test)
